libs/cms/src/lib/models/utils.ts

```diff
@@ -0,0 +1,3 @@
+/** Make sure that links are relative in case someone links to production directly */
+export const getRelativeUrl = (url: string) =>
+  new URL(url.trim(), 'https://island.is').pathname
```

Comment:
Could maybe remove all hosts? This is indeed the most common, even the only host in our urls. But what if someone accidentally adds `http://localhost` or `https://my-feature.dev01.devland.is`?

Reply:
Hmm, ok so if the host is similar to the check made here: 
![image](https://github.com/island-is/island.is/assets/43557895/dbe33f3d-b880-459c-b47a-2440fc281d3f)

Then we should remove it, to cover the cases where a link is perhaps pointing to dev? What if a link is supposed to be pointing to a feature deployed link, would we still want to overwrite it and make it relative?

Reply:
I noticed an error in my logic I was basically making external links be relative (by joinking their pathname) so I fixed that